command-line-flags-for-tidb-configuration.md

```diff
@@ -196,4 +236,14 @@ When you start the TiDB cluster, you can use command-line options or environment
 ## `--repair-list`
 
 + The names of the tables to be repaired in the repair mode.
-+ Default: `""`
\ No newline at end of file
++ Default: `""`
+
+## `-v`
```

Comment:
I think these flags comes from `160:    github.com/golang/glog v1.0.0 // indirect`, and they should not be included: they should not appear in tidb --help at all. This should be a bug.

`--temp-dir` and `--initialize-secure` sounds good to me.

Reply:
Looks like this affects all v6.x versions.

```
$ for b in $(ls .tiup/components/tidb/*/tidb-server | grep -v nightly); do echo "# $b"; $b -h 2>&1 | grep '\-vmodule'; done
# .tiup/components/tidb/v4.0.16/tidb-server
# .tiup/components/tidb/v5.0.6/tidb-server
# .tiup/components/tidb/v5.1.0/tidb-server
# .tiup/components/tidb/v5.1.3/tidb-server
# .tiup/components/tidb/v5.1.4/tidb-server
# .tiup/components/tidb/v5.2.4/tidb-server
# .tiup/components/tidb/v5.3.0/tidb-server
# .tiup/components/tidb/v5.3.1/tidb-server
# .tiup/components/tidb/v5.4.0/tidb-server
# .tiup/components/tidb/v5.4.1/tidb-server
# .tiup/components/tidb/v5.4.2/tidb-server
# .tiup/components/tidb/v6.0.0/tidb-server
  -vmodule value
# .tiup/components/tidb/v6.1.0/tidb-server
  -vmodule value
# .tiup/components/tidb/v6.1.1/tidb-server
  -vmodule value
# .tiup/components/tidb/v6.1.2/tidb-server
  -vmodule value
# .tiup/components/tidb/v6.2.0/tidb-server
  -vmodule value
# .tiup/components/tidb/v6.3.0/tidb-server
  -vmodule value
```

Could you file an issue for this? I'll remove it from this PR.

Reply:
Of course, pingcap/tidb#38857.